DemoViper/Sources/Services/Services.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  Services.swift
+//  DemoViper
+//
+//  Created by Su Van Ho on 13/8/19.
+//  Copyright © 2019 Nimble Co. Ltd. All rights reserved.
+//
+
+import Foundation
+
+final class Services {
+    static let shared: Services = Services()
+
+    func network() -> Network {
```

Comment:
I think here should be stored property with `let`. Every time you call `network()`, it will generate a new network instance 🤔

Reply:
👍